### PR TITLE
MinuteMediaPlus Bid Adapter: New Bid Adapter

### PR DESCRIPTION
**Pull request by uditalias**
## Type of change
- [x] New bidder adapter   

[prebid.github.io](https://github.com/prebid/prebid.github.io) Pull Request:
https://github.com/prebid/prebid.github.io/pull/4281

## Description of change
This is MinuteMediaPlus new bid adapter
Tests are included

- maintainer: hb@minutemedia.com
- test parameters for validating bids:
```
{
  bidder: 'minutemediaplus',
  params: {
    cId: '562524b21b1c1f08117fc7f9',
    pId: '59ac17c192832d0011283fe3',
    bidFloor: 0.0001,
    ext: {
      param1: 'loremipsum',
      param2: 'dolorsitamet'
    }
  }
}
```

### CODE REVIEW

**Comment by uditalias** [comment]
Hey @lksharma @jsnellbaker 
This PR is two weeks old. Please let me know if you have any updates or need something from my side.
Thanks.

**Review by lksharma — CHANGES_REQUESTED** [review]

File: modules/minutemediaplusBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+}
+
+function isBidRequestValid(bid) {
+  const params = bid.params || {};
+  return !!(extractCID(params) && extractPID(params));
+}
+
+function buildRequest(bid, topWindowUrl, sizes, bidderRequest) {
+  const { params, bidId, userId, adUnitCode, schain, mediaTypes } = bid;
+  let { bidFloor, ext } = params;
```
[the remark this answers is not in the document]
Comment (uditalias) [reply modules/minutemediaplusBidAdapter.js:59]: @lksharma Thanks
The bidder code changed to `mmplus`
The `bidFloor` is fine as it parsed on the server side

File: modules/minutemediaplusBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,291 @@
+import { _each, deepAccess, parseSizesInput, parseUrl, uniques, isFn } from '../src/utils.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER, VIDEO } from '../src/mediaTypes.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const GVLID = 918;
+const DEFAULT_SUB_DOMAIN = 'exchange';
+const BIDDER_CODE = 'minutemediaplus';
```
[the remark this answers is not in the document]
Comment (uditalias) [reply modules/minutemediaplusBidAdapter.js]: Bidder code changed to `mmplus`

**Review by lksharma — APPROVED** [review]
LGTM

**Pull request merged**